js_modules/dagster-ui/packages/ui-core/src/ui/GraphQueryInput.tsx

```diff
@@ -93,17 +95,30 @@ export const placeholderTextForItems = (base: string, items: GraphQueryItem[]) =
 
   if (seed === 0) {
     const example = ranked.sort((a, b) => b.outcount - a.outcount)[0];
-    placeholder = `${placeholder} (ex: ${example!.name}+*)`;
+    placeholder = `${placeholder} (ex: ${example!.name}${traversal('down')})`;
   } else if (seed === 1) {
     const example = ranked.sort((a, b) => b.outcount - a.outcount)[0];
-    placeholder = `${placeholder} (ex: ${example!.name}+)`;
+    placeholder = `${placeholder} (ex: ${example!.name}${traversal('down', 1)})`;
   } else if (seed === 2) {
     const example = ranked.sort((a, b) => b.incount - a.incount)[0];
-    placeholder = `${placeholder} (ex: ++${example!.name})`;
+    placeholder = `${placeholder} (ex: ${traversal('up', 2)}${example!.name})`;
   }
   return placeholder;
 };
 
+function traversal(direction: 'up' | 'down', levels?: number) {
+  if (featureEnabled(FeatureFlag.flagSelectionSyntax)) {
+    if (levels === undefined) {
+      return '+';
+    }
+    return direction === 'up' ? `+${levels}` : `${levels}+`;
+  }
+  if (levels === undefined) {
+    return '*';
+  }
+  return '+'.repeat(levels);
+}
+
```

Comment:
I wonder if there is also a callsite at `getQueryForVisibleAssets` that needs to be converted - that one powers the lineage tab and converts "depth=2" in the menu bar to "asset++"

Reply:
AH yeah I got that call site in https://github.com/dagster-io/dagster/pull/28037